src/documentation/pages/components/stat/examples/_portal/_background-icons.vue

```diff
@@ -0,0 +1,376 @@
+          .chi-stat__content
+            .chi-stat-metric
+              .chi-stat-metric__value
+                | {{ stat.value }}
+              .chi-stat-metric__title
+                | {{ stat.title }}
+            .chi-stat-submetric(v-if="stat.value2")
+              .chi-stat-submetric__value
+                | {{ stat.value2 }}
+              .chi-stat-submetric__title
+                | {{ stat.title2 }}
+            .chi-stat-background-icon(v-if="!stat.active")
+              chi-marketing-icon(:icon="stat.icon")
+            .chi-label__help(v-if="!stat.active")
+              button.chi-button.-icon.-xs.-flat(:id="`button-bg-icons-portal-${index}`" @click="togglePopover(`popover-${index}`)" aria-label="Help")
+                i.chi-icon.icon-circle-info-outline(aria-hidden="true")
+      chi-popover(v-for="index in stats.length-1" :key="index" :ref="`popover-${index}`" aria-modal="true" position="bottom" role="dialog" :reference="`#button-bg-icons-portal-${index}`")
+        .chi-popover__content
+          p.chi-popover__text
+            | Helpful information goes here.
+    </Wrapper>
+    <pre class="language-html" slot="code-webcomponent">
+      <code v-highlight="$data.codeSnippets.webcomponent" class="html"></code>
+    </pre>
+    <Wrapper slot="code-htmlblueprint">
+      <JSNeeded />
+      <pre class="language-html" >
+        <code v-highlight="$data.codeSnippets.htmlblueprint" class="html"></code>
+      </pre>
+    </Wrapper>  
+  </ComponentExample>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+
+@Component({
+  data: () => {
+    return {
+      stats: [
+        {
+          value: 'ALL',
+          title: 'Services',
+          active: true
+        },
+        {
+          value: '366',
+          title: 'Communications',
+          icon: 'platform-voice-unified-communications'
+        },
+        {
+          value: '932',
+          title: 'Hybrid & Cloud IT',
+          value2: '0',
+          title2: 'Recently Closed',
+          icon: 'platform-it-agility-hybrid-cloud'
+        },
+        {
+          value: '301',
+          title: 'Managed & IT Services',
+          value2: '6',
+          title2: 'Recently Closed',
+          icon: 'business-support-ticket'
+        },
+        {
+          value: '100',
+          title: 'Networking & Cloud IT',
+          icon: 'platform-adaptive-networking'
+        },
+        {
+          value: '2',
+          title: 'Security',
+          value2: '0',
+          title2: 'Recently Closed',
+          icon: 'platform-connected-security'
+        }
+      ],
+      exampleTabs: [
+        {
+          active: true,
+          id: 'webcomponent',
+          label: 'Web component'
+        },
+        {
+          id: 'htmlblueprint',
+          label: 'HTML blueprint'
+        }
+      ],
+      codeSnippets: {
+        webcomponent: `<div class="chi-stat">
```

Comment:
It should go on html + blueprint section, not in Web component

Reply:
in production the two tabs are enabled though? because of the marketing icon 🤔

Reply:
Yes